### ISSUE TITLE
WSOD upgrading T2 to T3

### ISSUE THREAD

**Issue by isumoc** [tripal/tripal#512]
The T3 upgrade docs were recently modified to add a warning about WSOD  (top of page here: http://tripal.info/tutorials/v3.x/upgrade/2.x-to-3.x ).

Does anyone know if T3 "Legacy Mode" installation is also affected by this ?

**Comment by spficklin** [comment]
Yes, it would be affected.  According to the comment about the WSOD, it is caused when Tripal v1 deprecated API functions are used.  if you never used Tv1 API deprecated functions in your custom modules then you're fine.  Are you experiencing a WSOD?

**Comment by isumoc** [comment]
Three weeks ago the 2x -> 3x was smooth and worked.  I have not kept up but will need to go back to look at what has changed since then. 

In my current upgrade attempt I have experienced multiple issues:

1. "registry_file" error from drush.
2. Hung process on "prepare chado" - to solve 1 and 2 I've moved back to an older VM so I know the state.
3. A network error with publications.  I am assuming the remote site contacted was offline as I cannot reproduce. Ignoring this one.
4. Upgrade completed but what appeared to be a WSOD.  I disabled all modules and extensions to sidestep this and get the upgrade to work.
5. Successful upgrade but Postgres dies.  Httpd process running up to 100% CPU until the network times out.  System completely offline.  Not a WSOD  - 100% BORKED. 😱 

I'll continue looking but perhaps some other users are having similar issues right now so I'll note this here.
